### PR TITLE
Comment reply review thumbnail

### PR DESCRIPTION
**Pull request by danglduy**
View, create, delete comments and replies. Add thumbnails to reviews show page.

### CODE REVIEW

**Comment by danglduy** [comment]
Merge this pull request first: #31

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  before_action :find_commentable
+  before_action :find_comment, only: [:update, :destroy]
+
+  def create
+    @comment = @commentable.comments
+      .build comment_params.merge user_id: current_user.id
+    if @comment.save
+      respond_to do |format|
+        format.html {
+          flash[:success] = "Your comment was successfully posted!"
```
Comment (chintk) [inline app/controllers/comments_controller.rb]: i18n

File: app/controllers/admin/reviews_controller.rb [hunk 2]
```diff
@@ -11,6 +11,8 @@ def show
 
   def new
     @review = @film.build_review
+    @reviewmodel = [:admin, @film, @review]
```
Comment (chintk) [inline app/controllers/admin/reviews_controller.rb]: viết luôn trong view

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (danglduy) [reply app/controllers/comments_controller.rb]: Done

File: app/controllers/admin/reviews_controller.rb (hunk 2 above)
Comment (danglduy) [reply app/controllers/admin/reviews_controller.rb]: Done

File: app/views/comments/_comment.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          </a>
+        <% end %>
+        <% if current_user.admin? %>
+          <%= link_to t("delete"), comment_path(comment), method: :delete,
+            remote: true, data: {confirm: t("delete_confirm")} %>
+        <% end %>
+      <% end %>
+    </div>
+  </div>
+  <% unless nil == comment.commentable.content %>
```
Comment (chintk) [inline app/views/comments/_comment.html.erb]: if .present?

File: app/views/comments/_comment.html.erb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <% if "Review" == comment.commentable_type %>
```
Comment (chintk) [inline app/views/comments/_comment.html.erb]: comment.commentable.is_a? Review

File: app/views/comments/_comment.html.erb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      <ul class="children" id="Comment-<%= comment.id %>-comments">
+        <% if comment.comments.exists? %>
+          <%= render partial: "comments/comment", collection: comment.comments %>
+        <% end %>
+      </ul>
+        <% if user_signed_in? %>
```
Comment (chintk) [inline app/views/comments/_comment.html.erb]: indent

File: app/views/comments/_comment.html.erb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+          <%= render partial: 'comments/reply_form', locals: {comment: Comment.new(user_id: current_user.id, commentable_type: "Comment", commentable_id: comment.id)} %>
```
Comment (chintk) [inline app/views/comments/_comment.html.erb]: commentable: comment

File: app/views/reviews/show.html.erb [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+      </div>
+    </div>
+  </div>
+  <div class="review-content">
+    <div class="container">
+      <div class="review-content-entry text-content">
+        <%= raw @review.content %>
+      </div>
+      <div class="review-share">
+        <span><%= t ".Share" %></span>
```
Comment (chintk) [inline app/views/reviews/show.html.erb]: key not good

**Comment by danglduy** [comment]
May quá cuối cùng cũng được merge T_T

**Pull request merged**